CMakeLists.txt

```diff
@@ -5,6 +5,8 @@ add_compile_definitions(_ENABLE_EXTENDED_ALIGNED_STORAGE)
 
 project(Wisp)
 
+option(ENABLE_UNIT_TEST "enables unit tests." OFF)
```

Comment:
We likely want to rename this to WISP_UNIT_TESTS = ON/OFF
So it doesn't collide.